language/stdlib/transaction_scripts/mint_lbr.move

```diff
@@ -1,11 +1,73 @@
 script {
 use 0x1::LibraAccount;
+// prover deps:
+use 0x1::Signer;
+use 0x1::Option;
 
 /// Mint `amount_lbr` LBR from the sending account's constituent coins and deposits the
 /// resulting LBR into the sending account.
+///
+/// # Events
+/// When this script executes without aborting, it emits three events:
+/// `SentPaymentEvent { amount_coin1, currency_code = Coin1, address_of(account), metadata = x"" }`
+/// `SentPaymentEvent { amount_coin2, currency_code = Coin2, address_of(account), metadata = x"" }`
+/// on `account`'s `LibraAccount::sent_events` handle where `amount_coin1` and `amount_coin2`
+/// are the components amounts of `amount_lbr` LBR, and
+/// `ReceivedPaymentEvent { amount_lbr, currency_code = LBR, address_of(account), metadata = x"" }`
+/// on `account`'s `LibraAccount::received_events` handle.
+///
+/// # Abort Conditions
+/// The following abort conditions have been formally specified and verified. See spec schema
+/// `LibraAccount::StapleLBRAbortsIf` for the formal specifications.
+///
+/// ## Aborts Caused by Invalid Account State
+/// * Aborts with `LibraAccount::EINSUFFICIENT_BALANCE` if `amount_coin1` is greater than sending
```

Comment:
I think we should document the category here (as the spec does) and leave the error reason to someone who actually wants to debug and uses @tzakian debug tool. The error reason may be brittle and is not part of the contract in my book. But definitely a difficult decision, so if you want to keep this, perhaps add a TODO that we need to settle the approach here.